### ISSUE TITLE
Unable to get library definition for neo4j-2.2.5.pom

### ISSUE THREAD

**Issue by peidong-hu** [cuba-platform/cuba-gradle-plugin#4]
Hi,

In my project, I am using dependency

compile group: 'org.springframework.data', name: 'spring-data-neo4j-rest', version:'3.4.6.RELEASE'
```
and get the following stacktrace in gradle deploy task when deploy  ,
Caused by: org.gradle.api.GradleException: Unable to get library definition for neo4j-2.2.5.pom
        at com.haulmont.gradle.libs.DependencyResolver.getLibraryDefinition(DependencyResolver.java:82)
        at com.haulmont.gradle.libs.DependencyResolver$getLibraryDefinition.callStatic(Unknown Source)
        at CubaDeployment$_deploy_closure2$_closure12.doCall(CubaDeployment.groovy:94)
        at org.gradle.api.specs.internal.ClosureSpec.isSatisfiedBy(ClosureSpec.java:32)
        at org.gradle.api.specs.AndSpec.isSatisfiedBy(AndSpec.java:46)
        at org.gradle.api.internal.file.AbstractFileTree$FilteredFileTreeImpl$1.visitFile(AbstractFileTree.java:180)
        at org.gradle.api.internal.file.collections.SingletonFileTree.visit(SingletonFileTree.java:42)
        at org.gradle.api.internal.file.collections.FileTreeAdapter.visit(FileTreeAdapter.java:110)
        at org.gradle.api.internal.file.AbstractFileTree$FilteredFileTreeImpl.visit(AbstractFileTree.java:172)
        at org.gradle.api.internal.file.CompositeFileTree.visit(CompositeFileTree.java:87)
        at org.gradle.api.internal.file.copy.CopySpecActionImpl.execute(CopySpecActionImpl.java:39)
        at org.gradle.api.internal.file.copy.CopySpecActionImpl.execute(CopySpecActionImpl.java:24)
        at org.gradle.api.internal.file.copy.DefaultCopySpec$DefaultCopySpecResolver.walk(DefaultCopySpec.java:648)
        at org.gradle.api.internal.file.copy.DefaultCopySpec.walk(DefaultCopySpec.java:458)
        at org.gradle.api.internal.file.copy.DelegatingCopySpecInternal.walk(DelegatingCopySpecInternal.java:236)
        at org.gradle.api.internal.file.copy.CopySpecBackedCopyActionProcessingStream.process(
```

**Comment by jreznot** [comment]
Hi,
Thank you for reporting the problem. We have created a YouTrack issue, see progress here: https://youtrack.cuba-platform.com/issue/PL-9821

**Comment by peidong-hu** [comment]
Thank jreznot.

Should I create issue on Youtrack or on github in the future for any finds about cuba?

Peidong

**Comment by jreznot** [comment]
It's ok to create issues and pull-requests here on github. Currently, we use YouTrack as internal bug tracker that is available for the community only in read-only mode.

**Comment by peidong-hu** [comment]
got it. thanks.

**Comment by jreznot** [comment]
Hi,
The fix is released in 6.6.5 version!
